build_scripts/OSX/MacOS_X_InstallGuide.md

```diff
@@ -108,7 +110,7 @@ Edit RetroShare.pro
 
     CONFIG += c++14 rs_macos11.1
 
-and then retroshare.pri
+Edit retroshare.pri
 
```

Comment:
it's surprising that the user would need to edit retroshare.pri. Cannot this be done in a specific MacOS section?

Reply:
not possible witthout edit, its something hardcoded gio developed that thing 
new pr her
https://github.com/RetroShare/RetroShare/pull/2797

Reply:
i had lots of issues in the passt we need edit always macos section to find sdk, to give via parameter not working